.github/workflows/tests.yml

```
@@ -69,6 +69,13 @@ jobs:
           cache: pip
           cache-dependency-path: requirements-tests.txt
       - run: pip install -r requirements-tests.txt
+      - name: Install external dependencies for 3rd-party stubs
+        run: |
+          DEPENDENCIES=$(python tests/get_external_stub_requirements.py)
+          if [ -n "$DEPENDENCIES" ]; then
+            echo "Installing packages: $DEPENDENCIES"
+            pip install $DEPENDENCIES
+          fi
```

(Same rationale as for the pyright job)

```suggestion
          fi
     - run: pip freeze --all
```

---

Thanks. Didn't notice that was missing.